lib/app.ts

```diff
@@ -7,9 +7,8 @@ import { CalculateReleaseByLabels } from "./features/calculate-release-by-labels
 const features = [CalculateReleaseByLabels, BlockIfMissingLabels];
 
 export = (app: Application) => {
-  const autobot = Autobot.start(app, features);
```

Comment:
This was really for development purposes. The bot by default runs on a lambda and thus only lives for the cycle of handling one webhook event. Sometimes when I'm testing I just run the raw probot instance. The `Autobot` instances is a bit stateful though, so this was causing some hard to debug issues. Now it just recreates the entire bot instance on every PR.